### ISSUE TITLE
The difference between Scenario-Bundles and Scenario-Comparison is not always clear to the user

### ISSUE THREAD

**Issue by jh-RLI** [OpenEnergyPlatform/oeplatform#1400]
## Description of the issue

As we released the Scenario-Bunldes with current release (v0.15.1) users can now use and see the feature on the OEP. While showing the new functionality the question was raised if the presented comparison is related to one of the Scenario-Bundles

## Ideas of solution

Describe possible ideas for solution and evaluate advantages and disadvantages.

## Workflow checklist
- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md)

**Comment by adelmemariani** [comment]
As the comparison contains study, input, and output datasets, we can consider it as scenario bundle comparison. But is not related to "one" scenario bundle. It is a comparison between scenario bundles.

**Comment by jh-RLI** [comment]
Yes, im not talking ontology style relations :P But you got a point. Maybe it is clear on a general level and we just need to add a simple tooltip that explains 1. how to start the comparison and 2. what it will compare (scenario bundles).
